lms/models/json_settings.py

```diff
@@ -18,25 +19,23 @@ class JSONSetting:
     """The group name that this setting is a part of."""
 
     key: str
-    """The key within the grouo that this setting is a part of."""
+    """The key within the group that this setting is a part of."""
 
     format: Any = str
     """An identifier to say what type of field this is."""
 
-    name: str | None = None
-    """An optional name for the field."""
+    default: Any = None
+
+    def __init__(self, compound_key: str, format_: Any = str, default=None):
+        self.group, self.key = compound_key.split(".")
+        self.format = format_
+        self.default = default
 
     @property
     def compound_key(self) -> str:
         """Get the group and key as a single value."""
         return f"{self.group}.{self.key}"
 
-    @property
-    def label(self) -> str:
```

Comment:
This was used in literally one setting. Let's remove it for now and revisit adding one later.